packages/ui/src/server/controllers/login.ts

```diff
@@ -41,7 +40,20 @@ export async function handleLogin(req: Request, res: Response) {
                     res,
                     response,
                     undefined,
-                    (headers) => removeSecureFlagIfOriginInsecure(req, headers),
+                    (headers: Record<string, string[]>) => {
+                        if (headers['set-cookie']) {
+                            headers['set-cookie'] = headers['set-cookie'].concat(
+                                headers['set-cookie'].map((item) =>
```

Comment:
All the cookies except YT_CYPRESS_COOKIE_NAME will be duplicated by `.map(...)` call, won't them?